src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Semantic/Models/RazorSemanticTokensLegend.cs

```diff
@@ -142,10 +144,11 @@ internal class RazorSemanticTokensLegend
             MarkupOperatorString,
             MarkupAttributeString,
             MarkupAttributeQuoteString,
-            MarkupTextLiteralString,
+            MarkupTextLiteralString, // 85
```

Comment:
Could we possibly add a large comment at the start of this array declaration this is a strictly ordered list and not to add out-of-order items?

Reply:
Not sure I follow. The numbered comments are just here to aid me in debugging things when I'm just getting a stream of numbers that I have to corollate back to this list.